tests/unit_tests/test_wave_06.py

```diff
@@ -143,7 +153,12 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    result == True
+    len(tai.inventory) == 3
+    len(jesse.inventory) == 3
+    tai.inventory == [item_a, item_b, item_f]
+    jesse.inventory == [item_d, item_e, item_c]
```

Comment:
It looks like the term `assert` is missing for these and the other tests in the rest of the file. Please make sure to remember to add `assert`!